extensions/ql-vscode/gulpfile.ts/typescript.ts

```diff
@@ -67,3 +67,13 @@ export function checkTypeScript() {
 export function watchCheckTypeScript() {
   watch("src/**/*.ts", checkTypeScript);
 }
+
+export function copyWasmFiles() {
+  // We need to copy this file for the source-map package to work. Without this fie, the source-map
+  // package is not able to load the WASM file because we are not including the full node_modules
+  // directory. In version 0.7.4, it is not possible to call SourceMapConsumer.initialize in Node environments
+  // to configure the path to the WASM file. So, source-map will always load the file from `__dirname/mappings.wasm`.
+  // In version 0.8.0, it may be possible to do this properly by calling SourceMapConsumer.initialize by
```

Comment:
Thanks for the extensive comment! It might be worth raising an issue for us to upgrade to v0.8.0 and seeing if we can get rid of the hack, once the PR is merged.

Reply:
Unfortunately, 0.8.0 hasn't actually been released yet and is only a beta. There doesn't [seem to be](https://github.com/mozilla/source-map/commits/master) a lot of activity in the repository, and it doesn't seem like a lot of progress is being made on a 0.8.0 release. I don't think an issue would help us here and I'm afraid it would get stale quite quickly. Hopefully dependabot will notify us when there's a new release, and we can re-evaluate then.

Reply:
Oh I didn't realise 0.8.0 wasn't out yet!